contrib/string_utilities.ads

```diff
@@ -0,0 +1,37 @@
+------------------------------------------------------------------------------
+--                        M A G I C   R U N T I M E                         --
+--                                                                          --
+--                       Copyright (C) 2021, AdaCore                        --
+--                                                                          --
+-- This library is free software;  you can redistribute it and/or modify it --
+-- under terms of the  GNU General Public License  as published by the Free --
+-- Software  Foundation;  either version 3,  or (at your  option) any later --
+-- version. This library is distributed in the hope that it will be useful, --
+-- but WITHOUT ANY WARRANTY;  without even the implied warranty of MERCHAN- --
+-- TABILITY or FITNESS FOR A PARTICULAR PURPOSE.                            --
+--                                                                          --
+-- As a special exception under Section 7 of GPL version 3, you are granted --
+-- additional permissions described in the GCC Runtime Library Exception,   --
+-- version 3.1, as published by the Free Software Foundation.               --
+--                                                                          --
+-- You should have received a copy of the GNU General Public License and    --
+-- a copy of the GCC Runtime Library Exception along with this program;     --
+-- see the files COPYING3 and COPYING.RUNTIME respectively.  If not, see    --
+-- <http://www.gnu.org/licenses/>.                                          --
+--                                                                          --
+------------------------------------------------------------------------------
+--  Subprograms to simplify transition to VSS for software that use Ada's
+--  standard string types.
+
+with VSS.Strings;
+
+package String_Utilities is
```

Comment:
let's use a more specific name than String_Utilities, which could clash with the namespace in client applications.

How about LSP.Utils.Conversions ?

Reply:
It is VSS, there is no LSP.* packages here :)

I think for LSP we should have more intelligent implementation, which at least:
 - try to load file with some "fallback" encoding in case of decoding failure
 - try to "recover" document in case of decoding failure

Reply:
ah yes silly me! I was thinking of

    VSS.Utils.Conversions

?